Manual/Releases/v4.19.0.lean

```diff
@@ -73,59 +98,84 @@ For this release, 420 changes landed. In addition to the 164 feature additions a
-  variable bound by a containing declaration.
+* [#7640](https://github.com/leanprover/lean4/pull/7640) implements the main logic for inheriting and overriding
+  autoParam fields in the `structure`/`class` commands, pending being
+  enabled in the structure instance notation elaborator. Adds term info to
+  overridden fields, so they now can be hovered over, and "go to
+  definition" goes to the structure the field is originally defined in.
 
-* [#7309](https://github.com/leanprover/lean4/pull/7309) fixes a bug where bv_decide's new structure support would
-  sometimes not case split on all available structure fvars as their type
-  was an mvar.
+* [#7717](https://github.com/leanprover/lean4/pull/7717) changes how `{...}`/`where` notation ("structure instance
+  notation") elaborates. The notation now tries to simulate a flat
+  representation as much as possible, without exposing the details of
+  subobjects.
+  This is a **breaking change**, see the PR description for further details and mitigation strategies.
 
-* [#7312](https://github.com/leanprover/lean4/pull/7312) implements proof term generation for `cooper_dvd_left` and its
-  variants in the cutsat procedure for linear integer arithmetic.
+* [#7742](https://github.com/leanprover/lean4/pull/7742) adds a feature to `structure`/`class` where binders without
+  types on a field definition are interpreted as overriding the type's
+  parameters binder kinds in that field's projection function. See the PR description for further details.
 
-* [#7314](https://github.com/leanprover/lean4/pull/7314) changes elaboration of `structure` parents so that each must be
-  fully elaborated before the next one is processed.
+### Library Updates
 
-* [#7315](https://github.com/leanprover/lean4/pull/7315) implements the Cooper conflict resolution in cutsat. We still
-  need to implement the backtracking and disequality case.
+* Developments in the async machinery;
+* Standardization of the integer division API;
+* Conversions between finite types;
+* API expansion of `BitVec` and tree maps;
+* Proofs of Bitwuzla rewrite rules;
+* Improvements to `List`/`Array`/`Vector`, as well as `HashMap` and `Int`/`Nat`.
+
+See the Library section below for details.
+
+### Other highlights
+
+* Documentation has been significantly expanded. See the Documentation section below for details.
+
+* [#7366](https://github.com/leanprover/lean4/pull/7366) adds server-side support for dedicated 'unsolved goals' and
+  'goals accomplished' diagnostics that will have special support in the
+  Lean 4 VS Code extension. The special 'unsolved goals' diagnostic is
+  adapted from the 'unsolved goals' error diagnostic, while the 'goals
+  accomplished' diagnostic is issued when a `theorem` or `Prop`-typed
+  `example` has no errors or `sorry`s. The Lean 4 VS Code extension
+  companion PR is at [leanprover/vscode-lean4#585](https://github.com/leanprover/vscode-lean4/pull/585).
```

Comment:
Maybe move this first, as the most user-visible change, and maybe even add a paragraph.

The companion change to the Lean 4 VS Code extension also adds new error and warning gutter decorations…

(See https://github.com/leanprover/vscode-lean4/pull/585 for details). This deserves highlighting :-)

Reply:
Thank you! Addressed in [f06e06b](https://github.com/leanprover/reference-manual/pull/442/commits/f06e06b1f07fb65be0087d69afe31487ea6d01ee)